### PR TITLE
Add very belated SC updates for 2020.

### PR DESCRIPTION
**Pull request by Yhg1s**
SC updates, distilled from meeting notes and email discussions, for January - October 2020.

### CODE REVIEW

**Comment by brainwane** [comment]
Thank you!

File: updates/2020-11-02-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+- SC will record a PyCon US 2020 Online talk May 11. A call for questions
+  [has been
+  posted](https://discuss.python.org/t/collecting-questions-for-a-steering-council-q-a/4010/2).
+
+
+## May recap:
+
+- The Steering Council discussed Eric's question about PEP 554 (multiple
+  Interpreters in the stdlib). It was decided Victor would loop in Antoine
+  and send SC motivations. - Steering Council discussed the Minimum Viable
```
Comment (gpshead) [inline updates/2020-11-02-steering-council-update.md:151]: missing paragraph/bullet break

File: updates/2020-11-02-steering-council-update.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+- Discussed candidates for GitHub project. Ezio was the preferred candidate.
+
+- Discussed PEP delegate process. Action item: Carol will amend PEP 1 with
+  language for proposed delegates to notify the steering council.
+
+- Approved Guido as delegate for PEP 618 (add optional length-checking to
+  zip).
+
+- Discussed the CZI grant process and potential projects:
```
Comment (gpshead) [inline updates/2020-11-02-steering-council-update.md:198]: link to an expansion of CZI

**Review by gpshead — COMMENTED** [review]
thanks for the writeup!

File: updates/2020-11-02-steering-council-update.md (hunk 1 above)
Comment (Yhg1s) [reply updates/2020-11-02-steering-council-update.md]: Fixed.

File: updates/2020-11-02-steering-council-update.md (hunk 2 above)
Comment (Yhg1s) [reply updates/2020-11-02-steering-council-update.md]: Done.

**Comment by vstinner** [comment]
Thank you very much @Yhg1s for writing down this document!!!

**Pull request merged**